src/tests/ftest/pool/simple_create_delete_test.py

```diff
@@ -47,7 +47,7 @@ def test_create(self):
         """
         Test basic pool creation.
 
-        :avocado: tags=pool,poolcreate,simplecreate
+        :avocado: tags=all,pool,smoke,full,tiny,simplecreate
```

Comment:
Was _full_ here meant to be _full_regression_?

Reply:
Done